helm_deploy/hmpps-book-a-video-link-api/values.yaml

```diff
@@ -35,7 +35,6 @@ generic-service:
     JAVA_OPTS: "-Xmx512m"
     SERVER_PORT: "8080"
     SPRING_PROFILES_ACTIVE: "logstash"
-    APPLICATIONINSIGHTS_CONNECTION_STRING: "InstrumentationKey=$(APPINSIGHTS_INSTRUMENTATIONKEY)"
```

Comment:
I missed taking this off the last PR.  Also trying this in the over arching values file.